alacritty/src/cli.rs

```diff
@@ -263,29 +265,34 @@ impl Options {
         }
 
         if let Some(lcr) = self.live_config_reload {
-            config.set_live_config_reload(lcr);
+            config.ui_config.set_live_config_reload(lcr);
         }
         config.shell = self.command.or(config.shell);
 
         config.hold = self.hold;
 
-        config.set_dynamic_title(config.dynamic_title() && self.title.is_none());
-        config.window.dimensions = self.dimensions.unwrap_or(config.window.dimensions);
-        config.window.title = self.title.unwrap_or(config.window.title);
-        config.window.position = self.position.or(config.window.position);
-        config.window.embed = self.embed.and_then(|embed| embed.parse().ok());
-
-        config.window.class.instance = self.class_instance.unwrap_or(config.window.class.instance);
-        config.window.class.general = self.class_general.unwrap_or(config.window.class.general);
-
-        config.debug.print_events = self.print_events || config.debug.print_events;
-        config.debug.log_level = max(config.debug.log_level, self.log_level);
-        config.debug.ref_test = self.ref_test || config.debug.ref_test;
-        config.debug.persistent_logging =
-            self.persistent_logging || config.debug.persistent_logging;
-
-        if config.debug.print_events {
-            config.debug.log_level = max(config.debug.log_level, LevelFilter::Info);
+        let ui_config = &mut config.ui_config;
+        ui_config.set_dynamic_title(ui_config.dynamic_title() && self.title.is_none());
+        let window_config = &mut ui_config.window;
```

Comment:
I'm not sure if extracting the `ui_config` into a variable is worth it? Seems to be infrequently used and short enough to just be accessed directly.

Reply:
It breaks formatting and makes it ugly.